src/App.js

```diff
@@ -7,8 +7,8 @@ import { Fragment } from "react";
 function App() {
   return (
     <Fragment>
-      <TodoList />
       <AddTodoForm />
+      <TodoList />
     </Fragment>
   );
```

Comment:
Where are you displaying the new todo? Refer to [Store "New Todo" in React State](https://github.com/Code-the-Dream-School/react-bald-eagle/wiki/Lesson-1.3#store-new-todo-in-react-state)